### DISCUSSION TITLE
pass class as props in CSS modules (V3)

### DISCUSSION THREAD

**Discussion by adammo94** [gatsbyjs/gatsby#30252] [Help]
Hello, I'd like to ask how can I pass class as props?

In V3 you migrated from `className={styles.sth}` to `className={sth}` which is fine.
However what can I do if I have following situation:

MyComponent:
`<Button modifier={'sth'}>click me</Button>`

Button:
``button onClick={onClick} className={`${container} ${modifier}`}>``

However this sth doesn't get a css module classname, just a regular `sth`, so Button doesn't recieve styles from modifier

**Comment by ccssmnn** [comment]
Are you referring to [this](https://www.gatsbyjs.com/docs/reference/release-notes/migrating-from-v2-to-v3/#css-modules-are-imported-as-es-modules) change?

From the code you provide it looks like you are passing the string `'sth'` instead of `sth` itself.
